cdci_data_analysis/flask_app/app.py

```diff
@@ -275,7 +275,11 @@ def run_analysis():
 def validate_schema(response):
     try:
         if dispatcher_strict_validate:
-            QueryOutJSON().load(response.json)
+            if response.is_streamed:
+                # TODO this should output a "default" response in case of download/js9 request
```

Comment:
you mean which output?

Reply:
It should create a default empty object, with default values, since no query output is generate by the dispatcher-app in case of download of files

Reply:
create where?

Reply:
It is just a default behavior in case no query output is produced by the dispatcher. We can also skip the validation for the specific case of download of files

Reply:
Well, this validation step here could validate more than QueryOutput, since all requests are passed through this method.
But testing download response by checking if {} matches QueryOutput is a bit misleading.

Indeed, I was suspecting some misunderstanding, since as of now, these default values are simply not used, and no default output is generated anywhere.

So it is better to ignore schema validation when there is  no schema.

Reply:
Then, for the moment, this whole PR is not necessary. 

Perhaps to mark this as a future improvement